proofs/lfsc/signatures/quantifiers_rules.plf

```diff
@@ -47,3 +47,29 @@
 
 (declare skolem_intro (! t term
                         (holds (= (skolem t) t))))
+
+; Beta-reduce t
+; We expect t to be of the form
+;   (apply (lambda v1 s1
+;     (apply (lambda v2 s2
+;     ...
+;     t2))
+;   t1))
```

Comment:
The locations of some closing parentheses look off to me. Did you mean?
```suggestion
;   (apply (lambda v1 s1
;     (apply (lambda v2 s2
;     ...)
;     t2))
;   t1)
```

Reply:
I find the updated version more confusing.  Note there are 2 opening parens on line 53, they are both closed on line 57.

Reply:
Nvm, the nesting was indeed wrong, I'll update.